lib/cirro_io_v2/responses/responses.rb

```diff
@@ -0,0 +1,97 @@
+module CirroIOV2
+  module Responses
```

Comment:
I am confused by the usage of a separate class as a response. 
Why the resource itself can not be the response itself?

What constraint or reason we need to go through the response class